deps/rabbitmq_prometheus/src/collectors/prometheus_rabbitmq_core_metrics_collector.erl

```diff
@@ -441,9 +442,9 @@ get_data(channel_metrics = Table, false) ->
                {messages_uncommitted, A4}, {acks_uncommitted, A5}, {prefetch_count, A6},
                {global_prefetch_count, A7}]}];
 get_data(queue_metrics = Table, false) ->
-    {Table, A1, A2, A3, A4, A5, A6, A7, A8, A9, A10, A11, A12, A13, A14, A15, A16} =
+    {Table, A1, A2, A3, A4, A5, A6, A7, A8, A9, A10, A11, A12, A13, A14, A15, A16, A17} =
```

Comment:
Instead of introducing a new metric, we could use the existing `status` field which all queue types have.

Reply:
Done!
In the previous change, there was a new entry `up` in the ETS table which was exposed as a Prometheus metric.
In the [latest commit](https://github.com/rabbitmq/rabbitmq-server/pull/2858/commits/a42ea8a3bbecf00f6e5d79a447963a46db2bf216), the entry `state` is re-used in the ETS table and a `state==running` metric (called `queues_up`)  is exposed as a Prometheus metric.